nilearn/datasets/atlas.py

```diff
@@ -1185,3 +1186,107 @@ def fetch_atlas_pauli_2017(version='prob', data_dir=None, verbose=1):
     return Bunch(maps=atlas_file,
                  labels=labels,
                  description=fdescr)
+
+
+
+def fetch_atlas_schaefer_2018(n_rois=100, yeo_networks=7, resolution_mm=1,
```

Comment:
I would somehow prefer the 400 as default. As this was our main parcellation in the paper.

Reply:
Thanks Alex. Changed to 400